src/python/bezier/curve.py

```diff
@@ -276,6 +276,38 @@ def evaluate_multi(self, s_vals):
         """
         return _curve_helpers.evaluate_multi(self._nodes, s_vals)
 
+    def evaluate_hodograph(self, s):
```

Comment:
cc @anntzer

I am (for now) just exposing the method that already exists. There is no equivalent `evaluate_hodograph_multi()`.

Somewhat related to this is #87.